src/reducers/console.js

```diff
@@ -1,8 +1,36 @@
-import {OrderedMap} from 'immutable';
+import inRange from 'lodash-es/inRange';
 
-import {ConsoleEntry, ConsoleError} from '../records';
+import {ConsoleState, ConsoleEntry, ConsoleError} from '../records';
 
-const initialState = new OrderedMap();
+const initialState = new ConsoleState();
+
+function updateConsoleForHistoryIndex(index, state) {
```

Comment:
Generally in reducers `state` is the first argument so for consistency let’s switch the order here?